vf_hi/site_core.php

```diff
@@ -533,7 +533,7 @@
 $Definition['Date.DefaultDateTimeFormat'] = '%B %e, %Y %l:%M%p';
 $Definition['Date.DefaultDayFormat'] = '%B %e';
 $Definition['Date.DefaultFormat'] = '%B %e, %Y';
-$Definition['Date.DefaultTimeFormat'] = '%l:%M%p';
+$Definition['Date.DefaultTimeFormat'] = 'g:iA;
```

Comment:
```suggestion
$Definition['Date.DefaultTimeFormat'] = 'g:iA';
```

Missing closing quote.